jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -224,6 +224,18 @@ YUI.add('user-profile', function() {
       this.close();
     },
 
+    /**
+      Set to a new, blank canvas with no model.
+
+      @method resetModel
+    */
+    resetModel: function() {
+      this.props.switchModel();
+      // XXX kadams54: What to do here?
+      this.props.dbEnvironmentSet('name', null);
```

Comment:
In the upcoming designs the breadcrumb will have an input which in which the user can specify the name before going through the flow. I'd imaging at this point we'd probably want to call it something like 'model1' (assuming there isn't a model1 already)